### ISSUE TITLE
Specifying --live-reload-port doesn’t seem to work

### ISSUE THREAD

**Issue by chasenlehara** [donejs/done-serve#70]
When I run this: `./node_modules/.bin/done-serve --develop --port 8080 --live-reload-port 9000`

I expect to be able to open the Network DevTools in Chrome, look at the WebSocket requests, and see the request url set to `ws://localhost:9000/`. Instead, it’s `ws://localhost:8012/`

__Environment:__

| Software                | Version
| ------------------ | -------
| donejs -V               | 1.0.1
| npm ls done-serve                   | 1.4.0
| npm ls steal-tools                   | 1.9.1
| node -v                  | 8.1.4

**Comment by NorthDecoder** [comment]
The issue is with the quick start documentation assuming the liberal use of localhost.
The guide could show an example of the following changes to ` package.json`:
```
"scripts": {
    //...
    "develop": "done-serve --develop --port 8080 ", // FROM
    "develop": "done-serve --develop --port 8080 --live-reload-port 8081", // TO
    //...
  },
```


```
  "steal": {
    //... other configuration values
   "liveReloadPort": "8081"// add this to configure the web page to look for this port
  },
```

**Comment by matthewp** [comment]
Yeah, so the problem is that `--live-reload-port` sets the port the server listens to, but not the port the client listens to.

If you think about it, it makes sense. The client runs in the browser, not in Node, so Node can't affect what port the client attempts to connect to.

The only way I can see this as possible is if you are using server-side rendering, we could maybe modifying the steal script tag and add `data-live-reload-port=PORT` to it or something.

**Comment by matthewp** [comment]
For those reading, to use a different live-reload port you have to specify it in 2 places, one in your index.stache steal script tag, change to:

```html
<script src="./node_modules/steal/steal.js" data-live-reload-port="9000"></script>
```

Or you could do it the way @NorthDecoder gives above.

And then in the cli do:

```shell
./node_modules/.bin/done-serve --develop --port 8080 --live-reload-port 9000
```

The comment above this one is discussing a way to prevent having to specify this in both places.
